### ISSUE TITLE
Is the model learning anything meaningful ?

### ISSUE THREAD

**Issue by UmarSpa** [li-plus/DSNet#1]
I have run your code for anchor-free model on canonical TVSum, as intructured in the README file. And I get the results similar to ones reported in the paper. To be precise, I get these numbers:

```
mean: 0.6160917484037374
split0: 0.624260622317302
split1: 0.5672167705637337
split2: 0.6329998280152374
split3: 0.6279028963330486
split4: 0.6280786247893654
```

But I have noticed that these best f-score numbers in each split are obtained right at the start of training (after a couple of of epochs). This can be observed in the following f-score vs epochs plot, where each colour corresponds to a separate TVSum split.

![image](https://user-images.githubusercontent.com/13482627/109317567-bda8d600-7844-11eb-85d1-78faf7b0e428.png)

As you can see the best f-score numbers are not much better than the f-scores obtained with randomly initialized weights of the model. This makes me if the model is indeed learning something meaningful. Any thoughts on this ?

**Comment by li-plus** [comment]
I think this is an overfitting problem because the TVSum dataset is really small with only 50 videos.

I further plot the training loss & training-set f-score & validation-set f-score vs training epochs on split 0 (other splits are similar). At epoch 0, the model is truly randomly initialized without any training.

![train300](https://user-images.githubusercontent.com/39846316/109914011-06034080-7cea-11eb-9185-00e7b0be1ba2.jpg)

As you can see, the training loss keeps descending, and the training-set f-score first increases and then stays at a high level (+8% than epoch 0). The validation-set f-score reaches the peak and then drops down.

Obviously, our model does learn something on the training-set, but 300 training epochs is too many on such a small dataset and causes overfit. We picked 300 just to be fair with VASNet (their training epochs are also 300). To relieve overfit, you may reduce the training epochs by specifying `--max-epoch 30` or use larger training datasets.
